nut.js

```diff
@@ -164,6 +164,10 @@ module.exports = function (db, precodec, codec, compare) {
 
       ltgt.toLtgt(opts, opts, encodeKey, precodec.lowerBound, precodec.upperBound)
 
+      // if these legacy values are in the options, remove them
+      delete opts.start
+      delete opts.end
+
```

Comment:
^ this is the thing that I dunno whether you want to do in `ltgt` or in this module